tests/sync/util.js

```diff
@@ -0,0 +1,194 @@
+const fs = require('fs-extra');
+const tmp = require('tmp-promise');
+const path = require('path');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+
+const syncFromDisk = require('../../sync/syncFromDisk');
+
+module.exports.course = {
```

Comment:
Unused?

Reply:
Unused - wrote that object to get a sense of how to structure the `CourseData` type.